.coderabbit.yaml

```
@@ -0,0 +1,245 @@
+reviews:
+  profile: assertive
+  request_changes_workflow: true
+  high_level_summary: true
+  poem: false
+  review_status: true
+  collapse_empty_files: true
+  auto_review:
+    enabled: true
+    drafts: false
+    base_branches: ["main", "develop", "staging"]
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)